src/main/menu.ts

```diff
@@ -316,5 +316,10 @@ export function buildMenu(windowManager: WindowManager) {
     appMenu.delete("mac");
   }
 
-  Menu.setApplicationMenu(Menu.buildFromTemplate([...appMenu.values()]));
+  return [...appMenu.values()];
+
+}
+
+export function buildMenu(windowManager: WindowManager) {
```

Comment:
Shouldn't we not call `initMenu` (the function that calls this function) on non-Macs?

Reply:
No harm to run it, since I reverted linux view to show native menus also.